webrender/src/lib.rs

```diff
@@ -48,7 +48,7 @@ extern crate cfg_if;
 extern crate lazy_static;
 #[macro_use]
 extern crate log;
-#[cfg(any(feature = "debugger", feature = "capture", feature = "replay"))]
+#[cfg(any(feature = "debugger", feature = "capture", feature = "replay", feature = "serialize_program"))]
```

Comment:
I wonder if we can just do `feature = "serde"` here

Reply:
I will update it in a next pull request.